vipps-login-api-faq.md

```diff
@@ -271,13 +273,44 @@ Vipps Login from phone number are reserved for such special cases and needs to b
+
+If you plan on using the scope delegatedConsents along with Vipps Login from phone number,
+you will need to supply some information on what consents you will retrieve and how.
+We will then tailor this screen to suit your needs. You can see what the flow looks like at
+[Vipps Login from phone number](vipps-login-from-phone-number-api-howitworks.md).
+Send an email to accessuserinfo@vipps.no and supply the following information:
+
+| Item                                                 | Description                                                                                                                                                                    | Example                                                                           | Comments                                                                                               |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| Top text                                             | Specifies the purpose of collecting the consents                                                                                                                               | "Merchant AS would like to send you tailored offers in several digital channels." |                                                                                                        |
+| Subset of consents you would like to ask for         | We offer a set of predefined consents. See [When using delegatedConsents, which consents does Vipps support?](#when-using-delegatedconsents-which-consents-does-vipps-support) | email, sms, digital, personal                                                     | If you want a consent type that we currently don't support, reach out to us at accessuserinfo@vipps.no |
+| Links to your membership terms and privacy statement | We need to check your terms to make sure the flow is not used to mislead or abuse the end user                                                                                 | www.merchant.com/termsandconditions <br /> www.merchant.com/privacystatement      |                                                                                                        |
+| Recipient to sign DPA                                | Vipps will function as a data processor and not have any ownership to the data involved. For more information, please visit merchant terms and conditions                      | recipient@merchant.com                                                            |                                                                                                        |
```

Comment:
```suggestion
| Recipient to sign DPA.                                | Vipps will function as a data processor and not have any ownership to the data involved. For more information, please visit [merchant terms and conditions](www.merchant.com/termsandconditions)                                            | [recipient@merchant.com](recipient@merchant.com)                                 |                                                                                                        |
```

Reply:
Do you define DPA anywhere above? If not, maybe spell it out here.

Reply:
This link is not yet ready